src/main.js

```diff
@@ -25,11 +25,11 @@ Vue.filter('stat', function (value) {
 })
 Vue.filter('aggressive', function (value) {
 	if (value == undefined) return 0;
-	if (value >= 1000) {
+	if (value >= 1000 && !store.rootGetters["settings/showFullValues"]) {
```

Comment:
Remember when I said 
![image](https://user-images.githubusercontent.com/16600313/88501567-de046f80-cf99-11ea-9bb3-9bdedbe6d940.png)

Well that was a lie. This should work though (actually tested)

```suggestion
	if (value >= 1000 && !store.getters["settings/showFullValues"]) {
```